runtime/android/core/src/org/xwalk/core/XWalkInitializer.java

```diff
@@ -11,13 +11,62 @@
 import org.xwalk.core.XWalkLibraryInterface.InitializationListener;
 
 /**
- * <code>XWalkInitializer</code> is a helper to initialize the Crosswalk environment as
+ * <code>XWalkInitializer</code> is a helper to initialize the Crosswalk Project runtime as
  * {@link XWalkActivity}. But unlike {@link XWalkActivity}, <code>XWalkInitializer</code>
```

Comment:
This will confuse XWalkInitializer belong to `XWalkActivity`. How about

```
<code>XWalkInitializer</code>· is a helper to initialize the Crosswalk Project runtime. But unlike 
{@link XWalkActivity}, <code>XWalkInitializer</code>
```